src/TConsole.cpp

```diff
@@ -2337,9 +2337,13 @@ std::pair<bool, QString> TConsole::setLabelToolTip(const QString& name, const QS
     return {false, QStringLiteral("label name \"%1\" not found").arg(name)};
 }
 
-void TConsole::createMapper(const QString& windowname, int x, int y, int width, int height)
+std::pair<bool, QString> TConsole::createMapper(const QString& windowname, int x, int y, int width, int height)
 {
     auto pW = mDockWidgetMap.value(windowname);
+    auto pM = mpHost->mpDockableMapWidget;
+    if (pM) {
+        return {false, "cannot create mapper. Do you already use a map window?"};
```

Comment:
:rotating_light: Too late for this PR - but this `QString` should be `QStringLiteral(...)` wrapped as it is a text for the Lua API and as such is NOT to be translated {if it was a GUI text then it should have a `tr(...)` wrapping} either way - it is not desirable to have raw C++ strings used to initialise `QString`s.

Reply:
https://github.com/Mudlet/Mudlet/pull/3456